portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -23,11 +23,11 @@
     <!-- jndi profile is defined here to prevent exceptions during war deployment -->
     <context-param>
         <param-name>spring.profiles.active</param-name>
-        <param-value>${dbconnector:jndi},${authenticate}</param-value>
+        <param-value>${dbconnector:dbcp},${authenticate}</param-value>
```

Comment:
How did changes to this file get into your PR?

Reply:
this was required for running webapp-runner.

Reply:
you could pass it as runtime argument `-Ddbconnector=dbcp`

Reply:
I mentioned above - the web-app-runner support has been merged into master and rc.  I think it should be removed from this PR and the PR should be rebased.